packages/frontend/src/graphql/mutations/retry-partial-step.ts

```diff
@@ -0,0 +1,7 @@
+import { gql } from '@apollo/client'
+
+export const RETRY_PARTIAL_STEP = gql`
+  mutation RetryPartialStep($input: RetryPartialStepInput) {
```

Comment:
nit: make input required
```suggestion
  mutation RetryPartialStep($input: RetryPartialStepInput!) {
```